CMakeLists.txt

```diff
@@ -96,6 +99,10 @@ elseif (SECURE_CHANNEL)
    set (MONGOC_ENABLE_CRYPTO_CNG 1)
 endif ()
 
+if (SYSTEM_CRYPTO_PROFILE)
```

Comment:
same as below. this has no affect when ENABLE_SSL=WINDOWS or DARWIN, so should be checked for